src/software/thunderscope/binary_context_managers.py

```diff
@@ -168,6 +168,7 @@ def setup_proto_unix_io(self, proto_unix_io):
             CostVisualization,
             NamedValue,
             PlayInfo,
+            SimulationStartedTrigger,
```

Comment:
If we don't use `LOG(VISUALIZE)` anymore, this will have to be moved to a separate `attach_unix_receiver` call where `from_log_visualize` is `False`

Reply:
Actually I just noticed that we shouldn't even add `SimulationStartedTrigger`, since this function is meant for `LOG(VISUALIZE)` calls from the full systems (i.e. the process that runs the AI), and not the simulator. But we send the `SimulationStartedTrigger` from the Simulator.